File: Store.BusinessLogic/Models/Filters/UserFilterModel.cs
using System.Collections.Generic;
using Store.BusinessLogic.Models.Base;

namespace Store.BusinessLogic.Models.Filters
{
    public class UserFilterModel : BaseFilterModel
    {
        public string SearchQuery { get; set; }

        public IList<bool> LockStatuses { get; set; }
    }
}

File: Store.View/src/app/shared/models/user/user.filter.model.ts
import { BaseFilterModel } from 'src/app/shared/models/base/base-filter.model';
import { UserLockStatus } from 'src/app/shared/enums';

export class UserFilterModel extends BaseFilterModel {
    public searchQuery?: string;
    public lockStatuses?: Array<UserLockStatus>;
}